src/frontend.md

```diff
@@ -52,7 +52,7 @@ src/
    type DocumentsById = Partial<Record<Document['id'], Document>>;
    ```
 
-1. Prefer specific types instead of common ones. It allows us to prevent more errors or development stage.
+2. Prefer specific types instead of common ones. It allows us to prevent more errors or development stage.
```

Comment:
I can't))
this is autoformatting in my IDE.
But it's right version, why do they all have to be with 1?

Reply:
> But it's right version, why do they all have to be with 1?

because markdown allows us to not care about numeration order by using 1 everywhere, it will be shown with right order eventually

> this is autoformatting in my IDE.

i believe you're able to disable formatting on save in your IDE
or just fix it outside of IDE